packages/@webex/plugin-meetings/package.json

```diff
@@ -42,7 +42,7 @@
   },
   "dependencies": {
     "@webex/common": "workspace:^",
-    "@webex/internal-media-core": "^0.0.17-beta",
+    "@webex/internal-media-core": "1.32.1",
```

Comment:
was this intentional?

Reply:
I am also curious. Because this looks like a version that'll get published to internal jFrog env.

Reply:
ya the main error was the media-core was not exporting the module properly